src/types/dict.rs

```diff
@@ -455,6 +455,99 @@ mod hashbrown_hashmap_conversion {
     }
 }
 
+#[cfg(feature = "indexmap")]
+#[cfg_attr(docsrs, doc(cfg(feature = "indexmap")))]
+mod indexmap_indexmap_conversion {
```

Comment:
Could this code actually please go in `src/conversions/indexmap.rs`? I realise you added alongside the `hashbrown` code but I think it's easier to find trait implementations / manage all optional features if we isolate them to new files.